packages/iframe-stamper/src/index.ts

```diff
@@ -145,29 +149,29 @@ export class IframeStamper {
 
     // This is populated once the iframe is ready. Call `.init()` to kick off DOM insertion!
     this.iframePublicKey = null;
+
+    this.messageChannel = new MessageChannel();
   }
 
   /**
    * Inserts the iframe on the page and returns a promise resolving to the iframe's public key
    */
   async init(): Promise<string> {
     this.container.appendChild(this.iframe);
+    this.iframe.addEventListener("load", () => {
+      // Send a message to the iframe to initialize the message channel
+      this.iframe.contentWindow?.postMessage("init", this.iframeOrigin, [
```

Comment:
why can `contentWindow` be null or undefined? Should we throw an error if `this.iframe.contentWindow` is falsy?

Reply:
In theory contentWindow should never be `null` or `undefined` but I always add optional chaining to short circuit and prevent an exception. Instead of this failing silently I will throw an exception here